pom.xml

```diff
@@ -240,6 +240,21 @@
       <optional>true</optional>
     </dependency>
 
+    <!-- from mg repository -->
+    <dependency>
+      <groupId>com.bergerkiller.bukkit</groupId>
+      <artifactId>TrainCarts</artifactId>
+      <version>1.20.6-v1</version>
+      <scope>provided</scope>
+      <optional>true</optional>
+    </dependency>
+    <dependency>
+      <groupId>com.bergerkiller.bukkit</groupId>
+      <artifactId>BKCommonLib</artifactId>
+      <version>1.20.6-v1</version>
+      <scope>provided</scope>
+      <optional>true</optional>
+    </dependency>
```

Comment:
This should not be necessary, as transitive dependencies should be declared in the pom of TrainCarts and so it gets pulled to without defining it.